tembo-operator/src/cloudnativepg/hibernate.rs

```diff
@@ -145,6 +146,60 @@ pub async fn reconcile_cluster_hibernation(cdb: &CoreDB, ctx: &Arc<Context>) ->
         }
     }
 
+    // Remove IngressRouteTCP route for stopped instances
+    let ingress_route_tcp_api = Api::namespaced(ctx.client.clone(), &namespace);
+    let prefix_read_only = format!("{}-ro-0", cdb.name_any().as_str());
+    if let Err(err) =
+        delete_ingress_route_tcp(ingress_route_tcp_api.clone(), &namespace, &prefix_read_only).await
+    {
+        warn!(
+            "Error deleting postgres ingress route for {}: {}",
+            cdb.name_any(),
+            err
+        );
+        return Err(Action::requeue(Duration::from_secs(300)));
+    }
+
+    let prefix_read_write = format!("{}-rw-0", cdb.name_any().as_str());
+    if let Err(err) = delete_ingress_route_tcp(
+        ingress_route_tcp_api.clone(),
+        &namespace,
+        &prefix_read_write,
+    )
+    .await
+    {
+        warn!(
+            "Error deleting postgres ingress route for {}: {}",
+            cdb.name_any(),
+            err
+        );
+        return Err(Action::requeue(Duration::from_secs(300)));
+    }
+
+    let prefix_pooler = format!("{}-poller-0", cdb.name_any().as_str());
+    if let Err(err) =
+        delete_ingress_route_tcp(ingress_route_tcp_api.clone(), &namespace, &prefix_pooler).await
+    {
+        warn!(
+            "Error deleting postgres ingress route for {}: {}",
+            cdb.name_any(),
+            err
+        );
+        return Err(Action::requeue(Duration::from_secs(300)));
+    }
+
+    let prefix_extra = format!("extra-{}-rw", cdb.name_any().as_str());
```

Comment:
Grabbed this from here 
https://github.com/tembo-io/tembo/blob/5b154ffbfc6f0a5d64e5f6a7949fede7c29f7806/tembo-operator/src/ingress.rs#L95